ott.LucyBot.yml

```diff
@@ -1,3 +1,5 @@
+basePath: /@VERSION@/doc/
```

Comment:
@bobby-brennan  
Although this change works for me perfect in our QA environment against multiple versions in different paths, I'm not sure how it will affect our formal generated version in https://ott.developer.kaltura.com/
WDYT?

Reply:
If you want a version dropdown like the one below, the versions will switch with the `?apiVersion=foo` query parameter. 

However, if you want the version to be in the path, you can create a separate specification for each version:
```yaml
specifications:
  ott_4:
    - openapi: ./ott4.openapi.json
  ott_3:
    -openapi: ./ott3.openapi.json
```

Then have separate routes for each version:

```yaml
routes:
  /v4/api-docs:
    title: Kaltura OTT API
    for: ott_4
  /v3/api-docs:
    title: Kaltura OTT API
    for: ott_3
```
The only catch is, you won't see the different versions in the dropdown like this:

![image](https://user-images.githubusercontent.com/7611973/38883746-dd6e49b2-423b-11e8-8d4b-dd3a3a0537d6.png)